### DISCUSSION TITLE
What's your favorite rock?

### DISCUSSION THREAD

**Discussion by atticusbaker** [earthlab-education/Earth-Analytics-AY24#130] [General]
I gotta know! I ask everyone and anyone this in Geology.

**Comment by lijo8146** [comment]
my favorite rock is the Little Elk Creek gneiss. my favorite mineral is labradorite.

**Reply by HeSapa9** [reply]
![Little Elk Gneiss](https://github.com/user-attachments/assets/46216ea7-b8cf-46f5-8e25-e1c311189f0a)

**Reply by nquarder** [reply]
@HeSapa9 & @lijo8146 is this this same gneiss we have in the office?! If so, can you remind us again how old this is!? Also where is it from and is there some cultural significance? Sorry for all the questions! :) -NQ

**Reply by lijo8146** [reply]
I'm not sure if Jim brought any to the office, but it is about 2.5 billion years old - some of the oldest material on the planet. There are some outcrops along Little Elk Creek in the Black Hills.

**Comment by HeSapa9** [comment]
Amphibolite
![7beb4d36-595f-44fb-9244-8f194c432629](https://github.com/user-attachments/assets/02eb0d76-99d1-46e4-aecc-5f0f2a4529d8)

**Reply by akpost21** [reply]
@HeSapa9 My dad is a geologist, so of course we had amphibolite countertops growing up- a very pretty stone!

**Comment by VictoriAragon** [comment]
Love obsidian

**Comment by eculler** [comment]
Well, it's not a rock itself, but I an fond of the rocks with mica in them -- we often see them in the Rocky Mountains up near Yellowstone:

![Boulder with Mica, Portsmouth Herald](https://www.seacoastonline.com/gcdn/authoring/2017/09/21/NPOH/ghows-SO-59b2d35e-fa8e-14bc-e053-0100007fedd1-259f9375.jpeg?width=1005&height=1276&fit=crop&format=pjpg&auto=webp)

**Comment by nquarder** [comment]
I like early 90's grunge/alternative rock ;)

**Reply by BioNCphilic** [reply]
Alice in Chains!

**Comment by allierosel** [comment]
Porphyritic basalt!

**Comment by ZimmZone** [comment]
I can't say I have a favorite, but I do like igneous rocks (rocks that formed from molten rock). I do know that Colorado's state rock is Yule Marble.
